content/platform/_index.md

```diff
@@ -53,46 +72,60 @@ sections:
 
         {{< video src="videos/jupyterlab.mp4">}}
```

Comment:
I am not seeing how to do this with the (hugo?) shortcode here, but if you add a `mute` on your videos, then they will actually auto play on all platforms. Right now the landing experience doesn't auto play the videos.